src/server/scripts/Kalimdor/BlackfathomDeeps/blackfathom_deeps.cpp

```diff
@@ -262,6 +236,6 @@ void AddSC_blackfathom_deeps()
 {
     new go_blackfathom_altar();
     new go_blackfathom_fire();
-    new npc_blackfathom_deeps_event();
+    RegisterCreatureAI(npc_blackfathom_deeps_event);
```

Comment:
should this be RegisterBlackfathomDeepsCreatureAI instead ?

Reply:
yup. will fix